### PR TITLE
Ussd Feedback

### PR DESCRIPTION
**Pull request by gsvr**
In addition to the questions in the spreadsheet for the post survey -> asking “How long did you travel for to get to the training you attended?"

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review. Should I remove all the opt-in and opt-out smsinbound functionality, or can I just leave it in?

File: src/feedback.js [hunk 1]
```diff
@@ -35,44 +36,169 @@ go.app = function() {
...
+                    return Q.all([
+                        self.im.contacts.save(self.contact),
+                        go.utils.post_feedback(self.im, self.contact, q_id, q_text_en,
+                                               choice.label, choice.value),
+                        self.im.outbound.send({
+                            to: self.contact,
+                            endpoint: 'sms',
+                            lang: self.contact.extra.lang,
+                            content: $("Testify! by sending an sms reply with your success story " +
+                                       "to this number.")
```
Comment (gsvr) [inline src/feedback.js]: @imsickofmaps I'm thinking of putting this text (well, an improved version) in the config so I can post the sms text in the wording for the smsinbound feedback post (q99) - what do you think?

Comment (imsickofmaps) [inline src/feedback.js]: :ok_hand:

Comment (imsickofmaps) [inline src/feedback.js]: `Thanks for providing feedback on your Emerging Leaders training. If you have a story about how it's changed your life in some way please reply to this SMS to tell us!`

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**